### PR TITLE
Change Equal in Test*Dao

### PR DESCRIPTION
**Pull request by duzhen1996**

### CODE REVIEW

File: smart-metastore/src/test/java/org/smartdata/metastore/dao/TestActionDao.java [hunk 1]
```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
```
Comment (littlezhou) [inline smart-metastore/src/test/java/org/smartdata/metastore/dao/TestActionDao.java]: use the previous license head

File: smart-metastore/src/test/java/org/smartdata/metastore/dao/TestActionDao.java [hunk 2]
```diff
@@ -52,12 +52,12 @@ public void testInsertGetAction() throws Exception {
     Map<String, String> args = new HashMap();
     args.put(CacheFileAction.FILE_PATH, "/test/file");
     ActionInfo actionInfo = new ActionInfo(1, 1,
-        "cache", args, "Test",
-        "Test", false, 123213213l, true, 123123l,
-        100);
-    actionDao.insert(new ActionInfo[] {actionInfo});
-    actionInfo = actionDao.getById(1l);
-    Assert.assertTrue(actionInfo.getCmdletId() == 1);
+            "cache", args, "Test",
```
Comment (littlezhou) [inline smart-metastore/src/test/java/org/smartdata/metastore/dao/TestActionDao.java]: 4 bytes alignment, previous version is right

File: smart-metastore/src/test/java/org/smartdata/metastore/dao/TestActionDao.java [hunk 3]
```diff
@@ -71,24 +71,24 @@ public void testUpdateAction() throws Exception {
     Map<String, String> args = new HashMap();
     args.put(CacheFileAction.FILE_PATH, "/test/file");
     ActionInfo actionInfo = new ActionInfo(1, 1,
-        "cache", args, "Test",
-        "Test", false, 123213213l, true, 123123l,
-        100);
+            "cache", args, "Test",
```
Comment (littlezhou) [inline smart-metastore/src/test/java/org/smartdata/metastore/dao/TestActionDao.java]: 4 bytes alignment, previous version is right

File: smart-metastore/src/test/java/org/smartdata/metastore/dao/TestActionDao.java [hunk 4]
```diff
@@ -71,24 +71,24 @@ public void testUpdateAction() throws Exception {
...
 
   @Test
   public void testGetNewDeleteAction() throws Exception {
     Map<String, String> args = new HashMap();
     args.put(CacheFileAction.FILE_PATH, "/test/file");
     ActionInfo actionInfo = new ActionInfo(1, 1,
-        "cache", args, "Test",
-        "Test", false, 123213213l, true, 123123l,
-        100);
+            "cache", args, "Test",
```
Comment (littlezhou) [inline smart-metastore/src/test/java/org/smartdata/metastore/dao/TestActionDao.java]: alignment

Comment (littlezhou) [inline smart-metastore/src/test/java/org/smartdata/metastore/dao/TestActionDao.java]: check alignment issue in other files also

File: smart-metastore/src/test/java/org/smartdata/metastore/dao/TestActionDao.java [hunk 5]
```diff
@@ -109,9 +109,9 @@ public void testMaxId() throws Exception {
...
```
Comment (littlezhou) [inline smart-metastore/src/test/java/org/smartdata/metastore/dao/TestActionDao.java]: check alignment issue in other files also

File: smart-common/src/main/java/org/smartdata/model/RuleInfo.java [hunk 6]
```diff
@@ -36,6 +36,42 @@
...
+    if (info == null || getClass() != info.getClass()) {
+      return false;
+    }
+
+    if (id != info.id
+        || submitTime != info.submitTime
+        || numChecked != info.numChecked
+        || numCmdsGen != info.numCmdsGen
+        || lastCheckTime != info.lastCheckTime
+        ) {
```
Comment (littlezhou) [inline smart-common/src/main/java/org/smartdata/model/RuleInfo.java]: move this line to the end of Line 61

File: smart-common/src/main/java/org/smartdata/model/RuleInfo.java [hunk 7]
```diff
@@ -36,6 +36,42 @@
...
+      return false;
+    }
+
+    if ((ruleText != null ? !ruleText.equals(info.ruleText) : info.ruleText != null)
+        || (state != null ? !state.equals(info.state) : info.state != null)
+        ) {
```
Comment (littlezhou) [inline smart-common/src/main/java/org/smartdata/model/RuleInfo.java]: move this line to the end of Line 67

File: smart-common/src/test/java/org/smartdata/model/TestFileInfo.java [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+    Assert.assertEquals(false, fileInfo.equals(fileInfo4));
+    Assert.assertEquals(false, fileInfo4.equals(fileInfo));
+
+    //Case 6:
+    FileInfo fileInfo5 = new FileInfo(" ", 1, 1, true, (short) 1, 1
+        , 1, 1, (short) 2, " ", " ", (byte) 1);
+    Assert.assertEquals(false, fileInfo.equals(fileInfo5));
+
+    //Case 7;
+    FileInfo fileInfo6 = new ChildFileInfo(" ", 1, 1, true, (short) 1, 1
```
Comment (littlezhou) [inline smart-common/src/test/java/org/smartdata/model/TestFileInfo.java]: Any reason to test this case?

File: smart-common/src/test/java/org/smartdata/model/TestFileInfo.java [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+    public ChildFileInfo(String path, long fileId, long length, boolean isdir, short block_replication, long blocksize, long modification_time, long access_time, short permission, String owner, String group, byte storagePolicy) {
+      super(path, fileId, length, isdir, block_replication, blocksize, modification_time, access_time, permission, owner, group, storagePolicy);
+    }
+  }
+
+  @Test
+  public void testEquals() throws Exception {
+    //Case 1:
+    FileInfo fileInfo = new FileInfo(" ", 1, 1, true, (short) 1, 1
+        , 1, 1, (short) 1, " ", " ", (byte) 1);
```
Comment (littlezhou) [inline smart-common/src/test/java/org/smartdata/model/TestFileInfo.java]: do not start the line with ','   move it to the end of the above line

Comment (littlezhou) [inline smart-common/src/test/java/org/smartdata/model/TestFileInfo.java]: check other lines/files also

**Comment by huafengw** [comment]
First please rebase with trunk

**Pull request merged**